README.md

```diff
@@ -7,7 +7,14 @@ A super simple JS library for playing sound effects and other audio.
 
 [Demo](https://sbrl.github.io/soundbox/example.html) (see also the [Real-World Usage](#real-world-usage) section!)
 
-Quick example:
+## Launch demo
```

Comment:
Since I've got a demo on GitHub Pages (see line 8 above this one), launching a local demo seems like a secondary thing. Perhaps we could refactor this slightly to suggest launching the online demo first, and then running it locally second?

The heading here draws the reader's attention away from the demo link above.

Come to think about it, shouldn't the demo work without a local HTTP server? Perhaps we could just launch Firefox instead? This sort of ties into my comment below.

Reply:
No it doesn't work locally without a http server.

Here's why.

I'm using Google Chrome Version 74.0 and when I downloaded the repo and tried to access `./example/index.html` I get this error network message in the Chrome Dev Tools console.

```
Access to script at 'file:///$PATH/soundbox/example/example.js' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, https.
```
When I remove the `type="module"` from `<script src="example.js" type="module"></script>` I get a syntax error because of the module imports inside the script.

This could work locally if we didn't use the module import feature.